### ISSUE TITLE
Incorrect request source in backend logs

### ISSUE THREAD

**Issue by bryk** [kubernetes/dashboard#272]
These are my logs. The source does not look right.
```
2016/01/20 13:36:45 Starting HTTP server on port 9090
2016/01/20 13:36:45 Creating API client for http://localhost:8080
2016/01/20 13:37:05 Incoming HTTP/1.1 GET /api/replicasets request from [
2016/01/20 13:37:05 Getting list of all replica sets in the cluster
2016/01/20 13:37:06 Outcoming response to [ with 201 status code
2016/01/20 13:37:08 Incoming HTTP/1.1 GET /api/events/kube-system/kubernetes-dashboard request from [
2016/01/20 13:37:08 Getting events related to kubernetes-dashboard replica set in kube-system namespace
2016/01/20 13:37:08 Incoming HTTP/1.1 GET /api/replicasets/kube-system/kubernetes-dashboard request from [
2016/01/20 13:37:08 Getting details of kubernetes-dashboard replica set in kube-system namespace
2016/01/20 13:37:09 Outcoming response to [ with 201 status code
2016/01/20 13:37:09 Found 9 events related to kubernetes-dashboard replica set in kube-system namespace
2016/01/20 13:37:09 Outcoming response to [ with 201 status code
2016/01/20 13:37:12 Incoming HTTP/1.1 GET /api/replicasets/pods/kube-system/kubernetes-dashboard request from [
2016/01/20 13:37:12 Getting list of pods from kubernetes-dashboard replica set in kube-system namespace with limit 0
2016/01/20 13:37:12 Incoming HTTP/1.1 GET /api/logs/kube-system/kubernetes-dashboard-ysffy request from [
2016/01/20 13:37:12 Getting logs from <nil> container from kubernetes-dashboard-ysffy pod in kube-system namespace
2016/01/20 13:37:12 Outcoming response to [ with 201 status code
2016/01/20 13:37:12 Outcoming response to [ with 201 status code
```

@maciaszczykm Can you take a look?

**Comment by maciaszczykm** [comment]
Yes, I will fix it today or tommorow.

**Comment by maciaszczykm** [comment]
This issue can be closed after/if pull request #283 will be merged.

**Comment by maciaszczykm** [comment]
@bryk This issue can be closed now.
